src/style/general.scss

```diff
@@ -33,7 +33,8 @@ a:hover {
 html,
 body {
 	margin: 0;
-	min-width: 320px;
+	// min-width: 320px;
+	min-width: 100%;
 	max-width: 100%;
```

Comment:
Why set both min-width and max-width to 100% ?
Should set width statically instead in that case.